client/src/components/button/TabButton.jsx

```
@@ -1,9 +1,10 @@
 import React from 'react';
-import { useNavigate } from 'react-router-dom';
 import styles from './TabButton.module.css';
 
-export default function TabButton({ content }) {
-  const navigate = useNavigate();
-
-  return <div onClick={() => alert(content)} className={styles.btn}>{content}</div>;
+export default function TabButton({ children, clickHandler }) {
+  return (
+    <div onClick={clickHandler} className={styles.btn}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)